library/kani_macros/src/sysroot/contracts.rs

```diff
@@ -276,6 +571,7 @@ fn rename_argument_occurences(sig: &syn::Signature, attr: &mut Expr) -> HashMap<
 ///
 /// ```rs
 /// #[kanitool::checked_with = "div_check_965916"]
+/// #[kanitool::replaced_with = "div_replace_965916"]
```

Comment:
It's just easier that way, because the infrastructure to parse these already exists in the compiler. This way it's also easier to add or remove them without having to change the parser every time. 

Once this is more stabilized we can make it prettier but these are also internal so idk if it's necessary.